bool FoodDelivered
bool ResidentEntertained
bool ResidentMedicated